pkg/apis/smith/v1/types.go

```diff
@@ -149,12 +156,21 @@ func (bc *BundleCondition) String() string {
 	return buf.String()
 }
 
+type PluginStatus struct {
+	Name    PluginName      `json:"name"`
+	Group   string          `json:"group"`
```

Comment:
use `metav1.GroupVersionKind` there (or its copy if you don't want to reuse `metav1` types in API)?

Reply:
That is what I did initially. But I didn't want to create a sub object for these fields just to re-use the struct.